plugin/gc-app/src/main/java/io/deephaven/app/GcPoolsPublisher.java

```diff
@@ -139,18 +139,21 @@ private void maybeFlushAndGrow(int initialSize, int poolSize) {
             consumer.accept(chunks);
         }
         chunkSize = Math.max(chunkSize, poolSize);
-        chunks = StreamToBlinkTableAdapter.makeChunksForDefinition(DEFINITION, chunkSize);
+        chunks = StreamChunkUtils.makeChunksForDefinition(DEFINITION, chunkSize);
     }
 
     private void flushInternal() {
         consumer.accept(chunks);
-        chunks = StreamToBlinkTableAdapter.makeChunksForDefinition(DEFINITION, chunkSize);
+        chunks = StreamChunkUtils.makeChunksForDefinition(DEFINITION, chunkSize);
     }
 
     public void acceptFailure(Throwable e) {
         consumer.acceptFailure(e);
     }
 
+    @Override
+    public void shutdown() {}
+
```

Comment:
I'm wondering, should it be our process here to notify downstream `consumer` when we are `shutdown`? Either, we can extend `StreamConsumer` interface in the same way, or we can do `consumer.acceptFailure(new PublisherShutdown())` or something. (Ditto for all other StreamPublisher impls).

Reply:
I could add documentation to that effect...
"This method should be invoked only by the eventual consumer, or by the constructing thread if no consumer will ever be registered".